Text-Based-Adventure/Text-Based-Adventure/main.swift

```diff
@@ -0,0 +1,459 @@
+/**/*/**/*/******,      .*##%%%%&&&%/,*//*,,//(//,,/*/%&&&&%(***,,,,,,,,,,,*/%%#(((/*////**//*,.,*,,*/#%%//(////*,,******,,,,..............
+/*//////(////(///*      .*%%%%%%%%%##((##/.....*(((###&&&%(****,,,,,,,,,,,,,,,*#%%%#(((((*.....,*////////(//////*********,*,,,,,,.,,.......
+////////(////(**/*        .,(%%%#//(((###%%%%%###%%%&&%(/**((/**,,,,,,,,,,,**/(/*/###/////(((((((//*****/###**********,,.....,,,...,.......
+////////////////**           ,#%(#%%###%%%%%%%%%%#####%%#**///(((/////////((///(#(////*/((///////***//(((/(#**,,,**,,,.....,,..............
+/*//////(//(/(///*          ..,(%%%%%%%%%%%%%%(((/*(#(#%%##*((/(/////*///(////**///**/*,*///(#####(//(/((//*,,,**,,,**,*/((//,,............
+/**/*/,**********,         .*,*#%%%%%%&%#/(/,,/((*,.**(%%%%%####((((((((((##((#(((/****,,,**,,,//#%#/////((/***,*,,,,,,,,,,/(*,,...........
+/****************,         .###%%%%%%&%(/*.. ..*,**.*,.*(%&%%%#################(///***,,,,,,,,,,,,(%%#///*////,,,,,,,,,,.,,**/,,...........
+/((////*(////(///*         ,#%##%%%%&%###///**///((**/#&&&&&%%###(((((((((#%%%#####***/***,,*/**/(//(#(///////,,,..,,.,,.,,,,*/*,..........
+//////*/////////** .... .   .*/###%%###((((//////(#%&%%&&&%%(/////********///(#%%###%%#///**///((///**//*****,,.,.....,,.,,,.,//,,.........
+/((**/**(//(/(*//*.*////////****(%%%########%%%%%###((%&%#(*,*(/***////***/(/**/**((///(((((((((((/////##*,,,,,..,,..,,,..,,,.,/*,,........
+///***,**************************/(#%%%%%%%%%%%%%%%###(((((#(/******///***/**,,**//////(/////****///(((//*,****,,,,,,,,,,,,,,..,/*,,.,,,,,,
+/*/*****/*********************/%%#%%%%%%%%&%**,**/(/,*(##%%#//////////**((/**/(((/**///******(%#(////((//##/,,,*,,,*,,,,,,,,*,.,/#*,,,,,..,
+/(((//**(//(/(/////***********(#####%%%%%&%(,..*,,*/,,/,,(#%%%%%%(((((((####(((/**,,,*,,,,,,,*(%%(//*/(((((*,,,,*,,*,,,,,,,,,*,,*(#,,..,,,,
+///////////////////***********/#%##%%%%%%%##//,**,*(*,/**(#&%%%##(((((((((#####(//*,,**,****/((%%#//**/((/*,,,,,,,,,,,,,,,,,,,,,,*#/,,,,..,
+////(/**(////(**//****************#######%#((#((((((/(#&&&&&&%##//////////(%%%%%##(******/((*,,***//*//****,,,,,,,,,,**,,,,.,,,,,,(#/,,,..,
+//////**///////*****************,,/*/%%/(##########(//#%%%(/**,,****,,**,*/###(/**////////******/#(***,,,,**,,,,,,,***.,,..,,,,,*%#,.,,,,
+/*/**/*//*/***********************####(#%%%%%%%%%%%%%%#(/(/**///*///*//(((*,,.,*((//////((/////(((**(((**,,,***,,*,,.,**,.,,,..,**,(%#*.,,,
+(///////(//(/(////****************##%(#%%%%%##%%%(////#((((/*/(((((/////((*,*//******///(#(////((((//(#**,,,***,,**,,,***..*,,,,,*,*(%#,,.,
+####((/((//////////*/***//*/****/*(##%%%%%%%%%&%(,,,,,/**##%%%//**********(((((//*,,**,,*/%%////((((//(,**,,,,**,,*,,,***,,,*,,,,,,,*%%/.,*
+""")
+    print("Okay \(enterName ?? "playa"), a \(howAdenturousAsAnInt) is adventurous enough to try the Vessel at Hudson Yards!!!......")
+    print("Ready to start? \"yes\" or \"no\"?")
+    
+    let readyGo = readLine()?.lowercased()
+    trophy.level = "Rockstar"
+    
+    
+    switch readyGo {
+    case "yes":
```

Comment:
You don't have an alternative for case "no", after I entered "no" it continued on.